spec/rubocop/cop/lint/deprecated_class_methods_spec.rb

```diff
@@ -3,38 +3,61 @@
 RSpec.describe RuboCop::Cop::Lint::DeprecatedClassMethods do
   subject(:cop) { described_class.new }
 
-  it 'registers an offense for File.exists?' do
-    expect_offense(<<-RUBY.strip_indent)
-      File.exists?(o)
-           ^^^^^^^ `File.exists?` is deprecated in favor of `File.exist?`.
-    RUBY
-  end
-
-  it 'registers an offense for ::File.exists?' do
-    expect_offense(<<-RUBY.strip_indent)
-      ::File.exists?(o)
+  context 'prefer `File.exist?` over `File.exists?`' do
+    it 'registers an offense for File.exists?' do
+      expect_offense(<<-RUBY.strip_indent)
+        File.exists?(o)
              ^^^^^^^ `File.exists?` is deprecated in favor of `File.exist?`.
-    RUBY
-  end
+      RUBY
+    end
+
+    it 'registers an offense for ::File.exists?' do
+      expect_offense(<<-RUBY.strip_indent)
+        ::File.exists?(o)
+               ^^^^^^^ `File.exists?` is deprecated in favor of `File.exist?`.
+      RUBY
+    end
 
-  it 'does not register an offense for File.exist?' do
-    expect_no_offenses('File.exist?(o)')
+    it 'does not register an offense for File.exist?' do
+      expect_no_offenses('File.exist?(o)')
+    end
   end
 
-  it 'registers an offense for Dir.exists?' do
-    expect_offense(<<-RUBY.strip_indent)
-      Dir.exists?(o)
-          ^^^^^^^ `Dir.exists?` is deprecated in favor of `Dir.exist?`.
-    RUBY
+  context 'prefer `Dir.exist?` over `Dir.exists?`' do
+    it 'registers an offense for Dir.exists?' do
+      expect_offense(<<-RUBY.strip_indent)
+        Dir.exists?(o)
+            ^^^^^^^ `Dir.exists?` is deprecated in favor of `Dir.exist?`.
+      RUBY
+    end
+
+    it 'registers an offense for ::Dir.exists?' do
+      expect_offense(<<-RUBY.strip_indent)
+        ::Dir.exists?(o)
+              ^^^^^^^ `Dir.exists?` is deprecated in favor of `Dir.exist?`.
+      RUBY
+    end
+
+    it 'does not register an offense for Dir.exist?' do
+      expect_no_offenses('Dir.exist?(o)')
+    end
   end
 
-  it 'auto-corrects File.exists? with File.exist?' do
```

Comment:
One final thing: It looks like these auto-correct specs were removed.

Reply:
Good catch! Added autocorrection specs for all 3 cases.